frontend/src/scenes/experiments/ExperimentView/StartDatePicker.tsx

```diff
@@ -0,0 +1,48 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { DatePicker } from 'lib/components/DatePicker'
+import { dayjs } from 'lib/dayjs'
+import { useState } from 'react'
+
+interface StartDatePickerProps {
+    currentStartDate: string
+    updateStartDate: (_: string) => void
+}
+
+export function StartDatePicker({ currentStartDate, updateStartDate }: StartDatePickerProps): JSX.Element {
+    const [isSelectorOpen, setIsSelectorOpen] = useState(false)
+
+    return (
+        <div>
+            {isSelectorOpen ? (
+                <DatePicker
+                    onSelect={(date: dayjs.Dayjs) => {
+                        updateStartDate(date.toISOString())
+                    }}
+                    showTime={false}
+                    open={true}
+                    showToday={false}
+                    mode="date"
+                    value={dayjs(currentStartDate)}
+                    disabledDate={(dateMarker) => {
+                        const now = new Date()
+                        return dateMarker.toDate().getTime() > now.getTime()
+                    }}
```

Comment:
I can't select today as the starting date. The left-hand side of the comparison here likely needs to be adjusted to the start of the day.

Reply:
Done!